Nighttime-Operations/Observing-Constraints/index.rst

```diff
@@ -22,7 +22,7 @@ Weather Constraints
 
 This page defines the different types of operational constraints.
 
-Operating conditions used to define the Rubin construction project are found in `LTS-54 <ls.st/lts-54>`__.
+Operating conditions used to define the Rubin construction project are found in LTS-54. 
```

Comment:
why did you remove the link?
you might need to use `https://ls.st/lts-54` if you're getting a broken link.

Reply:
👍